<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnTech::GuiQt::Resources::PaletteCentralWidget</class>
 <widget class="QWidget" name="UnTech::GuiQt::Resources::PaletteCentralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QToolButton" name="playButton">
       <property name="toolTip">
        <string>Play Animation</string>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/resources.qrc">
         <normaloff>:/icons/animation-play.svg</normaloff>:/icons/animation-play.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="animationFrameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="region"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
